modules/server/src/main/scala/navigate/server/NavigateEngine.scala

```diff
@@ -345,6 +348,19 @@ object NavigateEngine {
     )
 
     override def getGuideState: F[GuideState] = systems.tcsSouth.getGuideState
+
+    // This implementation in temporary
```

Comment:
Where does this come from in real life?

Reply:
It is composed from several EPICS channels in TCS.